### DISCUSSION TITLE
Mentoring

### DISCUSSION THREAD

**Discussion by AndreiExtr** [community/community#63178] [GitHub Education] [answered]
### Hi everyone,

Hello everyone!. Is it possible to communicate and learn from a mentor on the Github site? I would like to communicate/ exchange experience and knowledge on programming.

**Comment by humairshoukat** [comment, marked as answer]
Yep, you can contact a GitHub Campus Expert. Find you're nearest one here https://education.github.com/experts

**Reply by AndreiExtr** [reply]
Thanks😇
